src/libraries/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -234,6 +232,11 @@ public void GetFullPath_InvalidArgs()
                     { root + new string(Path.DirectorySeparatorChar, 3), root },
                 };
 
+                if (PlatformDetection.IsThreadingSupported){
```

Comment:
How is path normalization coupled with threading?

Reply:
I think this is more "if wasm".  If I understand it correctly, `Path.GetDirectoryName` is going to return "/" and going up a directory doesn't make a lot of sense.

Reply:
Agree. It should be `if (!PlatformDetection.IsBrowser)` then

Reply:
Done.